style.css

```diff
@@ -2854,3 +2848,113 @@ h4:hover .learn-more, h5:hover .learn-more, h6:hover .learn-more {
     color: #ff6760;
     font-size: 75%;
 }
+
+/* Fancybox */
```

Comment:
I was wondering... This "Fancybox" block is the styling of the "standalone" gallery, is it? Shouldn't this be nearly identical to the `.touchcarousel` styling? You could add CSS selectors to the already existing rules to clean this up and integrate it into the "regular" stylesheet. Makes it much more maintainable. :wink: :smiley: